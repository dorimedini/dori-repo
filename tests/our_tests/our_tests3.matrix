{
	matrix m(1,1) = [1];
	{
		if (m[0,0] == 1) {
			matrix m(2,2) = [1,2;3,4];
			print m;
		}
		foreach (x in m) {
		 	x = x+1;
		 	print "is it 2?";
			print x;
			print "is it [1]?";
			print m;
			m[0,0] = 0;
			print "is it 2?";
			print x;
			print "is it [0]?";
			print m;
		}
		print "is it [0]?";
		print m;
	}
}