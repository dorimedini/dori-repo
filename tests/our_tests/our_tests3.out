[1,2;3,4]
is it 2?
2
is it [1]?
[1]
is it 2?
2
is it [0]?
[0]
is it [0]?
[0]
